docs/security/secrets-store-csi/azure-key-vault.md

```diff
@@ -26,29 +26,13 @@ This document is adapted from the [Azure Key Vault CSI Walkthrough](https://azur
 
 {% include_relative install-kubernetes-secret-store-driver.md %}
 
-## Deploy Azure Key Store CSI
+## Deploy the Azure CSI driver
 
-1. Add the Azure Helm Repository
+1. Deploy the Azure CSI provider to ARO
 
     ```bash
-    helm repo add csi-secrets-store-provider-azure \
-      https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/charts
-    ```
-
-1. Update your local Helm Repositories
-
-    ```bash
-    helm repo update
-    ```
-
-1. Install the Azure Key Vault CSI provider
-
-    ```bash
-    helm install -n k8s-secrets-store-csi azure-csi-provider \
-      csi-secrets-store-provider-azure/csi-secrets-store-provider-azure \
-      --set linux.privileged=true --set secrets-store-csi-driver.install=false \
-      --set "linux.providersDir=/var/run/secrets-store-csi-providers" \
-      --version=v1.0.1
+    oc apply -n k8s-secrets-store-csi -f \
+      https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/deployment/provider-azure-installer.yaml
```

Comment:
This will leave the cluster in an unsupportable and un-upgradable state.  Unfortunately the default path of /etc/kubernetes puts the .sock file in a location that ContainerOS deems immutable and causes it to taint the node as unupgradable.